app/views/ride_mechanics/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= form_with url: "/ride_mechanics", method: :post, data: {turbo: false} do |form| %>
```

Comment:
Looks good. You can try playing with `form_with model:` to see if you can get the form to bind to the model.